src/pages/Contact/scenes/contact-sections-scene.tsx

```diff
@@ -0,0 +1,47 @@
+import { Grid, Typography, useTheme, Stack, Box } from "@mui/material";
+
+import { SocialLinksComponents } from "../components";
+
+export const ContactSectionScene = () => {
+  const theme = useTheme();
+  return (
+    <Grid item md={12}>
+      <Stack display="flex" direction="row" justifyContent="space-between">
+        <Stack>
+          <Typography
+            fontSize={"64px"}
+            fontWeight={"bold"}
```

Comment:
if the value is string remove {} 
eg: fontSize = {"24px"} instead make it fontSize = "24px" for more cleaner look